book_methods.rb

```diff
@@ -0,0 +1,9 @@
+class CreateBook
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
```

Comment:
- [ ] Requests for inputs should be handled by another method. Kindly extract these requests into another method destructure the answer as you have done with `make_person` in the `person_methods` file.